### PR TITLE
Pxe refs

### PR DESCRIPTION
**Pull request by dannf**
## Types of changes
- **Documentation Update**

## Description
Cleanups and adding troubleshooting info for legacy BIOS systems

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [x] Update the documentation if necessary

## Steps to Test This Pull Request
Just docs :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,9 @@ $ sudo cp /tmp/tmpxxx/ubuntu-installer/* /var/lib/tftpboot/
 Then your netboot server is ready to go if the corresponding DHCP is set up.
 
 ## Troubleshooting
-Please note that this tool is used in lab setup like
-[the setup described in this discourse post](https://discourse.ubuntu.com/t/netbooting-the-live-server-installer-via-uefi-pxe-on-arm-aarch64-arm64-and-x86-64-amd64/19240)
-to prepare a UEFI PEX server. It's suggested to check the details for your whole lab setup by reading the discourse
-post for trouble-shooting.
+For more details on setting up a PXE environment for x86 systems using a legacy BIOS, see [this discourse post](https://discourse.ubuntu.com/t/netbooting-the-live-server-installer/14510).
```
Comment (tai271828) [inline README.md]: I am thinking if we should point to this discourse post instead https://discourse.ubuntu.com/t/netbooting-the-server-installer-on-amd64/16620 , which is the "official" entry of "Netbooting the installer on amd64" of Ubuntu Server guide https://discourse.ubuntu.com/t/introduction/11322

https://discourse.ubuntu.com/t/netbooting-the-server-installer-on-amd64/16620 looks like an enhancement post of https://discourse.ubuntu.com/t/netbooting-the-live-server-installer/14510 to include UEFI for x86 machines.

Comment (tai271828) [reply README.md]: But the context is focusing on "legacy" vs. "UEFI", so it is ok for me to refer to https://discourse.ubuntu.com/t/netbooting-the-live-server-installer/14510

**Review by tai271828 — APPROVED** [review]
A nitpick comment inline. The overall looks good to me. Please merge anytime you want.

File: README.md (hunk 1 above)
Comment (dannf) [reply README.md]: Yeah, I'll change to the more official one. It doesn't hurt that it covers *both* UEFI and legacy. Thanks for the feedback!

**Pull request merged**